src/solvers/dgmulti/sbp.jl

```diff
@@ -221,7 +221,10 @@ function calc_volume_integral!(du, u, mesh::DGMultiMesh,
                 for id in nzrange(A_base, i)
                     j = rows[id]
                     u_j = u[j]
-                    A_ij = vals[id]
+
+                    # we use the negative of A_ij since A is skew symemtric, 
```

Comment:
```suggestion
                    # we use the negative of A_ij since A is skew-symmetric, 
```

Note the typo in symmetric

Reply:
Thanks!